test/unit.js

```diff
@@ -208,4 +208,25 @@ describe('check', function () {
       done();
     });
   });
+
+  it('Uses proxy from env vars', function (done) {
+
+    process.env.https_proxy = process.env.HTTPS_PROXY = 'http://127.0.0.1:8080';
+
+    var options = {
+      package: workingOptions.package,
+      shrinkwrap: workingOptions.shrinkwrap
+    };
+
+    Nock('http://127.0.0.1:8080')
+      .post('/check')
+      .reply(200);
+
+    Check(options, function (err, results) {
+
+      done();
```

Comment:
The reason is that lab will fail tests in that case, because Conf.proxy is always false. Lab ignores the last line in a conditional chain like that. In order to make lab happy any other way, we would have to pull in a .nsprc, modify it for that test, reload Conf, and change it back. While it looks strange, I thought this was a simpler solution that required fewer changes.